TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/commands/drive/Com_Rotate.java

```diff
@@ -0,0 +1,38 @@
+package org.firstinspires.ftc.teamcode.subsystems.commands.drive;
+
+import com.arcrobotics.ftclib.command.CommandBase;
+import com.arcrobotics.ftclib.hardware.RevIMU;
+import com.qualcomm.robotcore.util.ElapsedTime;
+
+import org.firstinspires.ftc.teamcode.subsystems.DriveSystem;
+
+public class Com_Rotate extends CommandBase {
+    private final DriveSystem driveSystem;
+    private final RevIMU imu;
+    private final double degrees;
+    //TODO: I got lazy add p control for imu later
+    //thx -past me
+    public Com_Rotate(DriveSystem subby, RevIMU revIMU, int degreesIn){
+        driveSystem = subby;
+        imu = revIMU;
+        //Question to Jackson I did this because I dont know what the imus max value is so I just
+        //said if it exceeds 360 subtract 360, does it have a max value is this needed I dont know
+        degrees = (degreesIn + imu.getHeading()) >= 360 ? (degreesIn + imu.getHeading())-360 : degreesIn + imu.getHeading();
```

Comment:
It should just be (-180, 180). If you want you can call `imu.getRotation2d().getDegrees()` which will wrap it for you.

Reply:
ok thx I was having trouble with this

Reply:
wait but degrees still has the potential to exceed 180 so I dont see how the getRotation2d would help

Reply:
Creating the Rotation2d object clamps the angle in the constructor to -180 to 180.

Reply:
Ahhh that's interesting thanks